src/sentry/static/sentry/app/utils/discover/fields.tsx

```diff
@@ -135,32 +135,66 @@ export const AGGREGATIONS = {
 
   // Tracing functions.
   p50: {
-    parameters: [],
+    parameters: [
+      {
+        kind: 'column',
+        columnTypes: ['duration'],
```

Comment:
For time time being, I would leave this as `duration` rather than all numeric types because columns like `stack.lineno` are also numeric and doesn't make sense to compute their percentiles. So we'll want to some additional work to filter out columns that we don't want here.